### ISSUE TITLE
Notebook Weekly Meeting 2023

### ISSUE THREAD

**Issue by echarles** [jupyter/notebook-team-compass#21]
Here is a record of the [agenda/minutes](https://hackmd.io/gl6x_SL0S_aYVn0Y0lSBpQ?both) for the Notebook weekly meeting, happening 30 minutes before the [weekly JupyterLab meeting](https://github.com/jupyterlab/jupyterlab#weekly-dev-meeting). 

- When: Wednesdays [8:30 am Pacific Time](https://dateful.com/convert/san-francisco-california?t=830am)
- Where: [`jovyan` Zoom](https://zoom.us/my/jovyan?pwd=c0JZTHlNdS9Sek9vdzR3aTJ4SzFTQT09)
- What: [Meeting notes](https://hackmd.io/gl6x_SL0S_aYVn0Y0lSBpQ?both) 

During the last meeting, we discussed posting the notes on a new issue in the team compass. Here is the record of these notes.

Feel free to comment here if you think the issue should be open somewhere else.

## Meeting Notes

- [30 August 2023](https://github.com/jupyter/notebook-team-compass/issues/21#issuecomment-1699591800)
- [23 August 2023](https://github.com/jupyter/notebook-team-compass/issues/21#issuecomment-1699591497)
- [09 August 2023](https://github.com/jupyter/notebook-team-compass/issues/21#issuecomment-1674351100)
- [26 July 2023](https://github.com/jupyter/notebook-team-compass/issues/21#issuecomment-1653025662)
- [04 January 2023](https://github.com/jupyter/notebook-team-compass/issues/21#issuecomment-1371170638)
- [11 January 2023](https://github.com/jupyter/notebook-team-compass/issues/21#issuecomment-1379888462)
- [18 January 2023](https://github.com/jupyter/notebook-team-compass/issues/21#issuecomment-1396550802)
- [25 January 2023](https://github.com/jupyter/notebook-team-compass/issues/21#issuecomment-1404816788)
- [01 February 2023](https://github.com/jupyter/notebook-team-compass/issues/21#issuecomment-1412507486)
- [08 February 2023](https://github.com/jupyter/notebook-team-compass/issues/21#issuecomment-1422965569)
- [15 February 2023](https://github.com/jupyter/notebook-team-compass/issues/21#issuecomment-1431700047)
- [22 February 2023](https://github.com/jupyter/notebook-team-compass/issues/21#issuecomment-1441472745)
- [01 March 2023](https://github.com/jupyter/notebook-team-compass/issues/21#issuecomment-1450484059)
- [08 March 2023](https://github.com/jupyter/notebook-team-compass/issues/21#issuecomment-1460504830)
- [15 March 2023](https://github.com/jupyter/notebook-team-compass/issues/21#issuecomment-1470307230)
- [22 March 2023](https://github.com/jupyter/notebook-team-compass/issues/21#issuecomment-1479850560)
- [29 March 2023](https://github.com/jupyter/notebook-team-compass/issues/21#issuecomment-1488896884)
- [05 April 2023](https://github.com/jupyter/notebook-team-compass/issues/21#issuecomment-1497831866)
- [12 April 2023](https://github.com/jupyter/notebook-team-compass/issues/21#issuecomment-1506366383)
- [19 April 2023](https://github.com/jupyter/notebook-team-compass/issues/21#issuecomment-1514992650)
- [03 May 2023](https://github.com/jupyter/notebook-team-compass/issues/21#issuecomment-1533432626)
- [17 May 2023](https://github.com/jupyter/notebook-team-compass/issues/21#issuecomment-1551815141)
- [24 May 2023](https://github.com/jupyter/notebook-team-compass/issues/21#issuecomment-1561705410)
- [31 May 2023](https://github.com/jupyter/notebook-team-compass/issues/21#issuecomment-1570523743)
- [07 June 2023](https://github.com/jupyter/notebook-team-compass/issues/21#issuecomment-1581162960)

**Comment by echarles** [comment]
## January 04, 2023

| name | org | username |
|------|-----|----------|
| Frederic Collonval | QuantStack | @fcollonval |
| Steven Silvester | MongoDB | @blink1073 |
| Eric Charles | Datalayer, Anaconda |@echarles |
|Rosio Reyes | Anaconda | @RRosio |
| Andrii Ieroshenko | AWS | @andrii-i |

Jeremy:
- Close the current meeting notes issue and open a new one for 2023? https://github.com/jupyter/notebook-team-compass/issues/5
- Do the same with the Hackmd notes?
    - We said during the meeting to keep the same hackm and simply prune it

**Comment by jtpio** [comment]
## January 11, 2023

| name | org | username |
|------|-----|----------|
| Eric Charles | Datalayer, Anaconda |@echarles |
| Jeremy Tuloup | QuantStack | @jtpio |
| Andrii Ieroshenko | AWS | @andrii-i | 
| Eric Gentry | Anaconda | @ericsnekbytes |
| Rosio Reyes | Anaconda | @RRosio |
| A. T. Darian | QuantStack | @afshin |
||||



- JupyerLab / Notebook 7 extensions
    - Summary of the discussion between @jtpio @ericsnekbytes @echarles
        - Context
            - Initial RetroLab's goal was just to have a simple UI, but not to support all the JupyterLab features. It had support for JupyterLab extensions since the beginning because it was able to reuse the same prebuilt extensions system
            - Then community members came with requests to support Left and Right side panels. This has been implemented in Notebook V7 (menus allow to show those left and right panels)
            - The extensions which do not interact with the main panel, or the status bar, or the top are should just work (e.g. resource usage is working)
            - We should extend the question to also JupyterLite, Voila, [Gator](https://github.com/mamba-org/gator)...
            - Key takeover is a need to improve the existing documentation to make this clearer to extension authors
        - Opportunities to add docs/other help for extension authors (nb7 + lab)
            - "Lab Extension tutorial" page: add a new section or link to another page (how to make your extension compatible with nb7 and lab)
            - "Extension points" page can be expanded
            - Can add in links from docs to extension-examples and jupyterlab-plugin-playground (latter can be used for additional teaching/examples)
        - We should track docs updates, AND nb7/lab extension compatibility in a meta issue perhaps, somewhere
- Governance update: https://jupyter.org/about

**Comment by jtpio** [comment]
## January 18, 2023

| name | org | username |
|------|-----|----------|
| Eric Charles | Datalayer, Anaconda |@echarles |
| Eric Gentry | Anaconda | @ericsnekbytes |
| A. T. Darian | QuantStack | @afshin |
| Rosio Reyes | Anaconda | @RRosio  |
| Andrii Ieroshenko | AWS | @andrii-i |
| Jeremy Tuloup | QuantStack | @jtpio |

- JupyterLab / Notebook 7 extensions
    - Defining INotebookShell as `optional` dependency in the plugin allows to handle the specifics of Notebook 7 - Question: How to ensure a plugin is not loaded at all?
- Rosio Reyes
    - Uffizzi integration PR request- Uffizzi seeks to replace Binder for pull request preview environments. [#6672](https://github.com/jupyter/notebook/pull/6672)
    - Split help-extension PR as is done in JupyterLab [#6700](https://github.com/jupyter/notebook/pull/6700)

**Comment by jtpio** [comment]
## January 25, 2023

| name | org | username |
|------|-----|----------|
| Eric Charles | Datalayer, Anaconda |@echarles |
| Eric Gentry | Anaconda | @ericsnekbytes |
| Rosio Reyes | Anaconda | @RRosio  |
| Andrii Ieroshenko | AWS | @andrii-i |

- Jeremy (will not attend the meeting):
    - New `7.0.0a11` pre-release: https://github.com/jupyter/notebook/releases/tag/v7.0.0a11
    - Add support for using the Settings Editor UI on the `/tree` page: https://github.com/jupyter/notebook/pull/6316
    - Try it on Binder with this gist: https://gist.github.com/jtpio/d368ab89cee5123ecee60683115e15f3
    - Use the `Settings > Settings Editor` menu entry to open it
- NbClassic
    - 0.5.0 release has been yanked, need a fix for the static and templates folder https://github.com/jupyter/nbclassic/issues/204
- Security Bug Bounty program

**Comment by echarles** [comment]
## February 01, 2023

| name | org | username |
|------|-----|----------|
| Eric Charles | Datalayer, Anaconda |@echarles |
| A. T. Darian | QuantStack | @afshin |
| Andrii Ieroshenko | AWS | @andrii-i |
| Eric Gentry | Anaconda | @ericsnekbytes |
| Rosio Reyes | Anaconda | @RRosio  |
| Frederic Collonval | QuantStack | @fcollonval |

- Announce upcoming Notebook 7 on Notebook 6.5
  - With a banner and/or console log on server startup?
  - Which other communication channels to use?
- Ensure Notebook 7 is released "some" time after JupyterLab 4
    - Minimum delay?
    - Minimum JupyterLab release? (4.0.1, 4.1.0)?
- Security Bug Bounty program
    - Which version?
    - Eric Charles, Rosio Reyes and Eric Gentry volunteer.
    - Any other volunteer?
- NbClassic (not discussed today)
    - NbClassic 0.5.1 is released (with CVE fixes)
    - Flaky javascript CI fixed
    - Disable pypy CI workflows?
    - Notebook not reconnecting to cell output in progress on refresh https://github.com/jupyter/nbclassic/issues/213

**Comment by echarles** [comment]
## February 08, 2023

| name | org | username |
|------|-----|----------|
| Jeremy Tuloup | QuantStack | @jtpio |
| Afshin T. Darian | QuantStack | @afshin |
| Zach Sailer | Apple | @Zsailer|
| Frederic Collonval | QuantStack | @fcollonval |
| Andrii Ieroshenko | AWS | @andrii-i |
| Nicolas Brichet | QuantStack | @brichet |
| Rosio Reyes | Anaconda | @RRosio |
| Eric Charles | Datalayer, Anaconda | @echarles |

- New Notebook 7 pre-release: https://github.com/jupyter/notebook/releases/tag/v7.0.0a12
- [News banner in NBClassic](https://github.com/jupyter/nbclassic/pull/215)
    - Messages to show?
        - `nbclassic`: `If you plan to upgrade to Notebook 7, read https://nbclassic.readthedocs.io/en/latest/nbclassic_notebook.html - ! 
Notebook 6 / NbClassic extensions will not work on Notebook 7.`
        - `notebook 6.5`: ?
        - `notebook 6.4`: ?
- Continuation of the discussion around ensuring Notebook 7 is released "some" time after JupyterLab 4
    - Minimum delay?
    - Minimum JupyterLab release? (4.0.1, 4.1.0)?
- The discussion on the 2 previous points has to happen on https://github.com/jupyter/nbclassic/issues/214
- Reply for Bounty program sent last week

**Comment by echarles** [comment]
## February 15, 2023

| name | org | username |
|------|-----|----------|
| Eric Charles | Datalayer, Anaconda | @echarles |
| Frederic Collonval | QuantStack | @fcollonval  |
| Afshin T. Darian | QuantStack | @afshin |
| Rosio Reyes | Anaconda | @RRosio |
| Jeremy Tuloup | QuantStack  | @jtpio  |
| Andrii Ieroshenko | AWS | @andrii-i |

- New Notebook 7 pre-release: https://github.com/jupyter/notebook/releases/tag/v7.0.0a13
    - Update to the latest JupyterLab 4.0.0a34
    - Accessibility fixes
    - Try it on Binder: https://gist.github.com/jtpio/d368ab89cee5123ecee60683115e15f3
- Communicate the upcoming Notebook 7
    - https://github.com/jupyter/nbclassic/issues/214
- Andrii: please add me to bug bounty program volunteer reviewers list
- Rosio: @frenzymadness opened issue for missing .po and .json translation files along with [PR](https://github.com/jupyter/notebook/pull/6728) for 6.5.x

**Comment by echarles** [comment]
## February 22, 2023

| name | org | username |
|------|-----|----------|
| Andrii Ieroshenko | AWS | @andrii-i |
| Rosio Reyes | Anaconda | @RRosio |
| Eric Gentry | Anaconda | @ericsnekbytes  |
| Zach Sailer | Apple | @Zsailer |
| Jeremy Tuloup | QuantStack | @jtpio |

Eric Charles (won't be able to participate)
- nbclassic 0.5.2 is released
- Have the Bounty program inputs been received by Jason (Grout)? 
    - Rosio: Followed-up with Jason, information about Notebook and NbClassic has been received and will be added.


Rosio Reyes
- mo and .json translation files: https://github.com/jupyter/notebook/pull/6728. 
    - Andrii: will check RU_ru
    - More details on translation process: https://github.com/jupyter/notebook/tree/6.5.x/notebook/i18n 
- Zmq message arrived on closed channel Issue https://github.com/jupyter/notebook/issues/6721 

Zach Sailer
- What's the story for Notebook v7, jupyter-server, and Binder?
    - https://discourse.jupyter.org/t/binder-switching-to-jupyter-server-by-default/18011 (courtesy of Nick)
- Today, notebook v7/lab is broken on Binder if Jupyter Server 2.0 is running underneath
    - Fix in progress: https://github.com/jupyter-server/jupyter_server/pull/1221

**Comment by echarles** [comment]
## March 01, 2023

| name | org | username |
|------|-----|----------|
| Jason Weill | AWS | @JasonWeill |
| Eric Gentry | Anaconda | @ericsnekbytes |
| Jeremy Tuloup | QuantStack | @jtpio |
| Eric Charles | Datalayer, Anaconda | @echarles |
| Rosio Reyes | Anaconda | @RRosio |
| Andrii Ieroshenko | AWS | @andrii-i |
| Nick Bollweg | Georgia Tech | @bollwyvl @nrbgt |
| Afshin T. Darian | QuantStack | @afshin |

nbextensions_configurator 
- nbclassic >=0.5 breaks jupyter_nbextensions_configurator https://github.com/Jupyter-contrib/jupyter_nbextensions_configurator/issues/151

Jason W: Notebook 7.0 release progress?

Jeremy:
- Latest pre-release: https://github.com/jupyter/notebook/releases/tag/v7.0.0a14
- 7.0 Release Plan: https://github.com/jupyter/notebook/issues/6307
- 7.0 issues: https://github.com/jupyter/notebook/issues?q=is%3Aopen+is%3Aissue+milestone%3A7.0
- Notebook talk accepted at JupyterCon

Rosio
- ZMQ Error temporary work around: https://github.com/jupyter/notebook/issues/6748

**Comment by echarles** [comment]
## March 08, 2023

| name | org | username |
|------|-----|----------|
| Eric Charles | Datalayer, Anaconda | @echarles |
| Frederic Collonval | QuantStack | @fcollonval |
| Eric Gentry | Anaconda | @ericsnekbytes | 
| A. T. Darian | QuantStack | @afshin |
| Rosio Reyes | Anaconda | |
| Andrii Ieroshenko | AWS | @andrii-i |
| Jeremy Tuloup | QuantStack | @jtpio |

- Eric
    - NbClassic 0.5.3 https://github.com/jupyter/nbclassic/releases/tag/v0.5.3 and Notebook 6.5.3 https://github.com/jupyter/notebook/releases/tag/v6.5.3 are released - The banner links to https://jupyter-notebook.readthedocs.io/en/latest/migrate_to_notebook7.html
    - nbclassic >=0.5 breaks jupyter_nbextensions_configurator https://github.com/Jupyter-contrib/jupyter_nbextensions_configurator/issues/151

- Jeremy
    - New `7.0.0a15` pre-release: https://github.com/jupyter/notebook/releases/tag/v7.0.0a15
    - Improvements pulled from JupyterLab:
        - configurable file name searcher in the file browser toolbar
        - file size column enabled by default (like the classic notebook)
    - Try it on Binder: https://gist.github.com/jtpio/d368ab89cee5123ecee60683115e15f3
    - Add support for `custom.css` up for grab if someone would like to work on it: https://github.com/jupyter/notebook/issues/6390
    - Try Notebook 7 with RTC enabled: https://gist.github.com/jtpio/0f3c67a6a1767360df0e79b52d74a5fe

**Comment by echarles** [comment]
## March 15, 2023

| name | org | username |
|------|-----|----------|
| Eric Charles | Datalayer, Anaconda | @echarles |
| A. T. Darian | QuantStack | @afshin |
| Frederic Collonval | QuantStack | @fcollonval |
| Eric Gentry | Anaconda | @ericsnekbytes |
| Andrii Ieroshenko | AWS | @andrii-i  |
| Rosio Reyes | Anaconda | @RRosio |
| Jeremy Tuloup | QuantStack  | @jtpio  |

- Eric Charles
    - nbclassic >=0.5 breaks jupyter_nbextensions_configurator https://github.com/Jupyter-contrib/jupyter_nbextensions_configurator/pull/152
    - Sync for JupyterCon talk
        - https://github.com/jupyter/notebook-team-compass/issues/22
- Jeremy (joined late)
    - New `7.0.0a16` pre-release: https://github.com/jupyter/notebook/releases/tag/v7.0.0a16
    - Try it on Binder: https://gist.github.com/jtpio/d368ab89cee5123ecee60683115e15f3
    - Try Notebook 7 with RTC enabled: https://gist.github.com/jtpio/0f3c67a6a1767360df0e79b52d74a5fe

- Rosio
    - nbgrader: Port to JLab 4 and Nb 7
      Best to ping Nicolas Brichet (@brichet) on https://github.com/jupyter/nbgrader/pull/1684
      
### Additional discussions

- Andrii: ChatGP demo used Jupyter Notebook: https://www.youtube.com/live/outcGtbnMuQ?feature=share&t=901

**Comment by echarles** [comment]
## March 22, 2023

| name | org | username |
|------|-----|----------|
| Eric Charles | Datalayer, Anaconda | @echarles |
|Andrii Ieroshenko | AWS | @andrii-i |
| Eric Gentry | Anaconda | @ericsnekbytes |
| Jason Weill | AWS | @JasonWeill |
| A. T. Darian | QuantStack | @afshin |
| Rosio Reyes | Anaconda | @RRosio |
| Jeremy Tuloup | QuantStack | @jtpio |
||||

- Jeremy
    - Notebook `7.0.0a17` release: https://github.com/jupyter/notebook/releases/tag/v7.0.0a17
    - Binder *without* RTC: https://gist.github.com/jtpio/d368ab89cee5123ecee60683115e15f3
    - Binder with RTC enabled: https://gist.github.com/jtpio/0f3c67a6a1767360df0e79b52d74a5fe

- Eric C.
    - nbclassic >=0.5 breaks jupyter_nbextensions_configurator impacting user
        - https://github.com/Jupyter-contrib/jupyter_nbextensions_configurator/pull/152#issuecomment-1479782967
    - Configurable banner display
        - https://github.com/jupyter/notebook/pull/6742#issuecomment-1479126043

- Jason, Andrii
    - Bugbash (group meet for bug discovery) on Monday March 27 
        - 09:00 PDT / 16:00 UTC / 17:00 BST / 18:00 CEST?

**Comment by echarles** [comment]
## March 29, 2023

| name | org | username |
|------|-----|----------|
| Eric Charles | Datalayer, Anaconda | @echarles |
| A. T. Darian | QuantStack | @afshin |
| Eric Gentry | Anaconda | @ericsnekbytes |
| Jeremy Tuloup | QuantStack | @jtpio |
| Jason Weill | AWS | @JasonWeill |
| Rosio Reyes | Anaconda | @RRosio  |
| William Stein | | |
| Frédéric Collonval | QuantStack | @fcollonval |

- Jeremy
    - Working on a Notebook 7 blog post, will share the draft soon
    - Should we release Notebook `7.0.0b0`?
        - Need a good changelog to summarize major user facing changes
        - Iterate on the migration guide: https://jupyter-notebook.readthedocs.io/en/latest/migrate_to_notebook7.html

- Eric C.
    - https://github.com/jupyter/nbclassic/pull/236 fixes the configurator issue reported in https://github.com/jupyter/nbclassic/issues/227
    - New release for nbclassic will be cut after https://github.com/jupyter/notebook/issues/6798 is implemented.

**Comment by echarles** [comment]
## April 05, 2023

| name | org | username |
|------|-----|----------|
| Eric Charles | Datalayer, Anaconda | @echarles |
| Andrii Ieroshenko | AWS | @andrii-i | 
| Jeremy Tuloup | QuantStack | @jtpio |
| Nicolas Brichet | QuantStack | @brichet |
| Afshin T. Darian | QuantStack | @afshin |
| Eric Gentry | Anaconda | @ericsnekbytes |
| Rosio Reyes | Anaconda | @RRosio  |

- Eric C.
    - `show_banner` trait for Notebook 6.5 is merged
    - nbclassic 0.5.4 (for marked compatiblity) and 0.5.5 (for jupytext edit handler) are released.
    - When do we cut nbclassic 1.0.0? https://github.com/jupyter/nbclassic/issues/238

- Notebook 7 / NbClassic
    - Add the "Open with Classic Notebook" entry in the interface dropdown if NBClassic is available https://github.com/jupyter/notebook/issues/6746

- Communication Plan
    - https://github.com/jupyter/notebook-team-compass/issues/24
    - When to release Notebook 7 ?
        - How much time between JupyterLab4 and Notebook 7?
        - Plan 1 hour next week to further discuss and take a decision?

- Jeremy
    - Notebook `7.0.0b0` is available:
        - https://github.com/jupyter/notebook/releases/tag/v7.0.0b0
        - Try it on Binder: https://gist.github.com/jtpio/d368ab89cee5123ecee60683115e15f3
        - Try it on Binder with Real Time Collaboration enabled: https://gist.github.com/jtpio/0f3c67a6a1767360df0e79b52d74a5fe
    - Documentation improvements:
        - https://github.com/jupyter/notebook/pull/6813
        - Split the migration guide into multiple pages
        - Add page dedicated to the new features in Notebook 7
        - Update screencasts and screenshots
        - Please review, help and submit more improvements!
    - (draft) blog post to encourage users to test Notebook 7 beta:
        - https://github.com/jupyter/notebook-team-compass/issues/24
    - Help Wanted:
        - https://github.com/jupyter/notebook/issues/6692
        - https://github.com/jupyter/notebook/issues/6390

**Comment by yuvipanda** [comment]
Re: Notebook 7, one suggestion coming from the education world is to do it after Mid-may 2023 - many universities are on summer / winter break after that, and this could prevent some unfortunate surprises right around finals times.

**Comment by echarles** [comment]
> Re: Notebook 7, one suggestion coming from the education world is to do it after Mid-may 2023 - many universities are on summer / winter break after that, and this could prevent some unfortunate surprises right around finals times.

Thx @yuvipanda for highlighting that very important milestone. We'll make sure JupyterCon (May 10-12, 2023) is not the ultimate and unique target.

**Comment by jtpio** [comment]
FYI there is a dedicated issue to discuss the 7.0 release plan: https://github.com/jupyter/notebook/issues/6307. It would be great to keep the conversation there so feedback and other comments are not lost between weekly meeting notes. Thanks!

**Comment by echarles** [comment]
## April 12, 2023

| name | org | username |
|------|-----|----------|
| Eric Charles | Datalayer, Anaconda | @echarles |
| Andrii Ieroshenko | AWS | @andrii-i | 
| Afshin T. Darian  | QuantStack | @afshin | 
| Eric Gentry | Anaconda  | @ericsnekbytes | 
| Jeremy Tuloup | QuantStack  | @jtpio  | 
| Rosio Reyes | Anaconda | @RRosio  | 

- NbClassic Configurator
    - https://github.com/Jupyter-contrib/jupyter_nbextensions_configurator/issues/151

- Communication Plan
    - https://github.com/jupyter/notebook-team-compass/issues/24
    - The large user base is today Notebook 6.
    - Notebook 7 and JupyterLab 4 are built on the same technical stack, so they could be released at the same time.
    - However having some time between the release would allow to get more feedback and fix the bug reports
    - Feedback from the education world on https://github.com/jupyter/notebook-team-compass/issues/21#issuecomment-1497850313 asking to release Notebook 7 after Mid-may 2023 to avoid potential issues for the exams.
    - Not all countries have exams at the same time.
        - Some US schools have exams until mid June
    - Education typically depend on JupyterHub with Docker images which are immutable. If the Notebook versino is pinned, this would not break, if the version is not pinned, there is a risk that extensions will break.
    - Fedora distribution is already packaging Notebook 7 alpha.
    - Still room for making the current migration documentation better, with potential usage of Wiki or GitHub discussion. Documentation will mitaigate and issues.
    - Releasing Notebook 7 after JupyterCon would harm the product launch.
    - NbClassic could be shipped with Notebook 7 to respect the JEP spirit, possibility for some initial period. There are counter arguments for that listed on https://github.com/jupyter/notebook/issues/6746#issuecomment-1496948227 - User will see those options in the menu. Attention should be put on the existing drop-down to launch nbclassic from notebook 7 (is that not too visible for the user?). There is today on drop-down to go from nbclassic to notebook 7 (this can be added).
    - We could log the extensions that do not have equivalent.
    - We need a list of things that need to happen before the release of Notebook 7. A list exists on an NbClassic issue but should be formalized on the Notebook repo.
    - We don't know JupyterLab 4 issues that will come back, there for the "security period" before releasing Notebook 7 is can not be defined today.
    - JupyterLab Release Candidate initially planned on April 8th has been delayed.
    - The blog post should invite users to try the beta version of Notebook 7. The content will be further worked-out in a HackMd.

- JupyterLab 4 [`release blocking` issues](https://github.com/jupyterlab/jupyterlab/issues?q=is%3Aopen+is%3Aissue+milestone%3A4.0.0+label%3A%22tag%3ARelease+Blocker%22)

**Comment by echarles** [comment]
## April 19, 2023

| name | org | username |
|------|-----|----------|
| Eric Charles | Datalayer, Anaconda | @echarles |
| A. T. Darian | QuantStack | @afshin |
| Eric Gentry | Anaconda | @ericsnekbytes |
| Andrii Ieroshenko | AWS | @andrii-i |
| Frederic Collonval | QuantStack | @fcollonval |
| Nicolas Brichet | QuantStack | @brichet |
| Rosio Reyes | Anaconda | @RRosio  | 

- Eric C.
    - NbClassic logo changes (Fedora distribution)
    - Add an option to disable the news / Notebook 7 migration banner https://github.com/jupyter/nbclassic/issues/242
        - Need a new `notebook_shim` release with https://github.com/jupyter/notebook_shim/pull/29
        - notebook configurator needs a new release https://github.com/Jupyter-contrib/jupyter_nbextensions_configurator/issues/151#issuecomment-1507259596 - Jurgen has been contacted for that.
        - Should Notebook 7 depend on NbClassic? https://github.com/jupyter/notebook/issues/6746#issuecomment-1496948227

- Jeremy (not attending):
    - At PyConDE & PyData Berlin this week
    - Gave a talk on Notebook 7: https://github.com/jupyter/notebook-team-compass/issues/22#issuecomment-1514673533
        - Quick poll of the room (~50 people): approximately 50% using the classic notebook, the other 50% JupyterLab
        - No major worries or questions about extension compatibility
        - Not many questions asked at the end of presentation
        - Questions about whether RTC will work in Notebook, which suggests some folks will like to have this feature available in Notebook
    - Anyone had time to look at the draft blog post and make some edits? When can we publish it?
        - Updates given by Rosio and Eric C.
        - https://github.com/jupyter/notebook-team-compass/issues/24

- Andrii:
    - Attempting to convert a notebook with pasted image to PDF fails with an error, nbconvert issue #1977 https://github.com/jupyter/nbconvert/issues/1977 

- Eric Gentry
    - Trying to improve developer experience
    - Identifying best next steps
    - Tool checks...

**Comment by jtpio** [comment]
## April 26, 2023

| name | org | username |
|------|-----|----------|
| Eric Charles | Datalayer, Anaconda | @echarles |
| Jeremy Tuloup | QuantStack | @jtpio |
| Andrii Ieroshenko | AWS | @andrii-i |
| A. T. Darian | QuantStack | @afshin |
| Eric Gentry | Anaconda | @ericsnekbytes |
| Rosio Reyes | Anaconda | @RRosio  | 

- Eric Gentry
    - PR for updated NbClassic homepage README
        - https://github.com/jupyter/nbclassic/pull/246
    - PR for more info on 6 to 7 migration docs
        - https://github.com/jupyter/notebook/pull/6850
- Jeremy
    - Make sure to convey the right message for the Jupyter Notebook project
    - Notebook talk at JupyterCon: https://github.com/jupyter/notebook-team-compass/issues/22
    - Install `nbclassic` via the `jupyter` metapackage instead of the `notebook` package?
        - The `jupyter` metapackage is very popular
        - Users often "pip install jupyter" when they are told to "Install Jupyter"
        - This would help address the concerns of a hard dependency on `nbclassic` (getting the old UI while you installed the new one)
- Andrii 
    - What is Notebook 7 Release Plan? JupyterLab 4.0.0 waits to resolve all blocking issues and will release an 4.0.0 RC, 2 weeks later 4.0.0
        - Jeremy: for reference: https://github.com/jupyter/notebook/issues/6307
            - Andrii: was checking if there are any updates 
    - Notebook 7 bugbash during JupyterCon sprints? Notebook 7 talk team should have received an email with invitation to propose a sprint

**Comment by echarles** [comment]
## May 03, 2023

| name | org | username |
|------|-----|----------|
| Eric Charles | Datalayer, Anaconda | @echarles |
| Jeremy Tuloup | QuantStack | @jtpio |
| Afshin T. Darian | QuantStack | @afshin | 
| Andrii Ieroshenko | AWS | @andrii-i | 
| Nicolas Brichet | QuantStack | @brichet | 
| Frederic Collonval | QuantStack | @fcollonval |
| Rosio Reyes | Anaconda | @RRosio  | 
| Eric Gentry | Anaconda | @ericsnekbytes |

- Eric C.
    - Will cut NbClassic 1.0.0 this week

- SSC Notebook update to the EC
    - Asked to report the Notebook project status. This is the proposal TBD
        - Notebook 7 built on JupyterLab 4
            - Debugger, RTC, Theming…
            - Supports the “labextensions”
        - NbClassic
            - Jupyter Server extension
            - Support the “nbextensions”
        - Notebook 6
            - Maintenance mode (security fixes)
            - 6.5: Reuses the nbclassic user interface
        - Migration guide
            - https://jupyter-notebook.readthedocs.io/en/latest/migrate_to_notebook7.html 

- Jeremy
    - `7.0.0b1` release: https://github.com/jupyter/notebook/releases/tag/v7.0.0b1
    - Based on JupyterLab 4.0.0rc0
    - Try it on Binder: https://gist.github.com/jtpio/d368ab89cee5123ecee60683115e15f3
    - When can we release Notebook `7.0.0rc0`?

- Andrii
    - [`jupyter/notebook #6847`](https://github.com/jupyter/notebook/pull/6847) is ready for review but UI tests fail after rebasing on master and "Bot please update playwright snapshots" doesn't work, bot might be broken https://github.com/jupyter/notebook/pull/6847

- Blog post and Talk
    - Talk is being worked-out
    - Blog post still need to be reviewed and updated

- Triaging
    - Started triaging 7.0 issues from newest, started triaging from newest issues, triaged up to and including issue #6847

**Comment by jtpio** [comment]
## May 17, 2023

| name | org | username |
|------|-----|----------|
| Jeremy Tuloup | QuantStack | @jtpio |
| Andrii Ieroshenko | AWS | @andrii-i |
| Eric Charles | Datalayer, Anaconda | @echarles |
| Afshin T. Darian | QuantStack | @afshin |
| Eric Gentry | Anaconda | @ericsnekbytes |

- Jeremy:
    - JupyterLab 4 is released :tada: https://github.com/jupyterlab/jupyterlab/releases/tag/v4.0.0
    - Post JupyterCon action items for Notebook 7:
        - Blog post: x (blogpost itself: https://jupyter-notebook.readthedocs.io/en/latest/migrate_to_notebook7.html)
        - Identify "Release Blocker" issues?
        - Would like to get the following in before making an RC:
            - https://github.com/jupyter/notebook/pull/6841
            - https://github.com/jupyter/notebook/issues/6688
            - https://github.com/jupyter/notebook/issues/6692
        - More to triage: https://github.com/jupyter/notebook/issues?q=is%3Aopen+is%3Aissue+milestone%3A7.0
        - Help on these issues highly appreciated!
        - Notebook 6 extension shim can be developed and explored in parallel:
            - https://github.com/jupyter/notebook/issues/6394
            - Doesn't have to be ready before 7.0 final
- Eric Gentry:
    - Ecosystem docs improvement PRs are in-progress (Jupyter.org meta docs, Notebook docs)
        - Meta docs updates at
https://github.com/jupyter/jupyter/pull/682
        - Notebook docs updates at
https://github.com/jupyter/notebook/pull/6877
        - Meta glossary updates at
https://github.com/jupyter/jupyter/pull/681

- Eric Charles
    - 0.5.6: pep517 build fails  https://github.com/jupyter/nbclassic/issues/251

- (time permitting) nick
    - trove classifiers https://github.com/jupyter/notebook/issues/6875
    - simple PR, but sometimes takes a whilele https://github.com/pypa/trove-classifiers/pull/72

- Andrii
    - Update on Interface / Open in... menu and dynamic detection of NbClassic work: https://github.com/jupyter/notebook/pull/6847#issuecomment-1516811779
    - Snapshot update bot is broken: https://github.com/jupyter/notebook/issues/6859

**Comment by andrii-i** [comment]
## May 24, 2023

| name | org | username |
|------|-----|----------|
| Eric Gentry | Anaconda | @ericsnekbytes |
| Andrii Ieroshenko | AWS | @andrii-i |
| Rosio Reyes | Anaconda | @RRosio  | 

- Andrii
    - https://github.com/jupyter/notebook/issues/6307#issuecomment-1557529038 : Currently there are 59 open issues is the Notebook 7 milestone [link](https://github.com/jupyter/notebook/milestone/45). If we want to release Notebook 7 in the next couple of weeks, we should identify “release blockers”. Should we create “release blocker” tag and review list of open Notebook 7 issues identifying release blockers during Jupyter Notebook Meeting on Wednesday (this can take up the entire meeting)?
- Eric Gentry
    - Ecosystem docs updates are in (multiple PRs), would like to get these merged in soon, then ITERATE/improve
        - Notebook
            - https://github.com/jupyter/notebook/pull/6877
        - Lab
            - https://github.com/jupyterlab/jupyterlab/pull/14568
        - `Jupyter.org` meta-docs
            - https://github.com/jupyter/jupyter/pull/682
    - Can add governance links/info to subproject docs, link to meta project


- (Andrii) During May 24, 2023 Notebook and JupyterLab meetings we triaged Notebook 7.0 milestone bugs to identify release blockers: 

Before triage, 20 notebook 7.0 bugs needed triage. After triage, 4 notebook 7.0 bugs still need triage

Triaged notebook 7 bugs:
https://github.com/jupyter/notebook/issues/4761
https://github.com/jupyter/notebook/issues/6384
https://github.com/jupyter/notebook/issues/6385
https://github.com/jupyter/notebook/issues/6386
https://github.com/jupyter/notebook/issues/6387
https://github.com/jupyter/notebook/issues/6419
https://github.com/jupyter/notebook/issues/6553
https://github.com/jupyter/notebook/issues/6671
https://github.com/jupyter/notebook/issues/6697
https://github.com/jupyter/notebook/issues/6711
https://github.com/jupyter/notebook/issues/6790
https://github.com/jupyter/notebook/issues/6792

Non-triaged notebook 7 bugs:
https://github.com/jupyter/notebook/issues/6800
https://github.com/jupyter/notebook/issues/6822
https://github.com/jupyter/notebook/issues/6821
https://github.com/jupyter/notebook/issues/6819

**Comment by echarles** [comment]
## June 7, 2023

| name | org | username |
|------|-----|----------|
| Jeremy Tuloup | QuantStack | @jtpio | 
| Eric Gentry | Anaconda | @ericsnekbytes |
| Frederic Collonval | QuantStack | @fcollonval |
| Eric Charles | Datalayer, Anaconda | @echarles |
| Andrii Ieroshenko | AWS | @andrii-i |
| Afshin T. Darian | QuantStack | @afshin |
| Rosio Reyes | Anaconda | @RRosio  |

- Andrii: 
    - WCAG 2.1 accessibility issue: prioritization, notebook 7 blocking https://github.com/jupyter/notebook/issues/6800
        - We should consider an issue as blocking if it prevents someone currently using Notebook 6 to migrate to Notebook 7. Either because there is a big regression, or because something is missing in Notebook 7.
    - Should we aim to close all blocking issues and release by end of the next week?
        - We should look at Notebook JEP 79 and what is promised for Notebook 7 release within it https://jupyter.org/enhancement-proposals/79-notebook-v7/notebook-v7.html
            - Critical extensions:
                - nbgrader
                - RISE (not done)
                - Jupytext
                - IPython Parallel (filebrowser tab)
            - Extensions whose functionality will be built into Notebook v7:
                - Table of Contents
                - Collapsible headings
            - Major new features in Notebook v7:
                - Debugger
                - Real-time collaboration
                - Theming and dark mode
                - Internationalization
                - Improved Web Content Accessibility Guidelines (WCAG) compliance
                - Support for many JupyterLab extensions, including Jupyter LSP (Language Server Protocol) for enhanced code completions
            - Timeline: 2022

**Comment by jtpio** [comment]
## June 14, 2023

| name | org | username |
|------|-----|----------|
| Jeremy Tuloup | QuantStack | @jtpio | 
| Andrii Ieroshenko | AWS | @andrii-i |
| Afshin T. Darian | QuantStack | @afshin |
| Eric Charles | Datalayer, Anaconda | @echarles |
| Eric Gentry | Anaconda | @ericsnekbytes |
| Rosio Reyes | Anaconda | @RRosio  |

- Jeremy:
    - Customize the shell layout with the settings: https://github.com/jupyterlab/jupyterlab/pull/11614
    - Should help support more lab extensions that add widgets to the `main` area
    - Once finished and merged, release the RC?
    - Need to list and track other tasks that should be done in parallel:
        - Timeline and updates: https://github.com/jupyter/notebook/issues/6307
        - Blog post: https://github.com/jupyter/notebook-team-compass/issues/24

**Comment by andrii-i** [comment]
Update on notebook 7 blogpost: https://github.com/jupyter/notebook-team-compass/issues/24#issuecomment-1593518927

**Comment by jtpio** [comment]
## June 21, 2023

| name | org | username |
|------|-----|----------|
| Jeremy Tuloup | QuantStack | @jtpio | 
| Andrii Ieroshenko | AWS | @andrii-i |
| Eric Gentry | Anaconda | @ericsnekbytes |
| A. T. Darian | QuantStack | @afshin |
| Rosio Reyes | Anaconda | @RRosio  |
| tonyfast |  | @tonyfast  |
| Luciano Resende | Apple | lresende|
| Amola Hinge | Apple | amolahinge
| Ian Dong | Apple | iand-23 |

- Jeremy:
    - `7.0.0rc0` is released: https://github.com/jupyter/notebook/releases/tag/v7.0.0rc0
    - Still possible to fix issues in the `7.0.x` milestone: https://github.com/jupyter/notebook/issues?q=is%3Aopen+is%3Aissue+milestone%3A7.0.x
    - Anyone working on https://github.com/jupyter/notebook/issues/6394 ?
    - Close old issues in the notebook repo:
        - https://github.com/jupyter/notebook-team-compass/issues/27
        - Help wanted :pray:  

- Andrii 
    - Blogpost draft: https://hackmd.io/6kExBNFcRZCIpBtQDrB0ow?both
    - Comment about last feedback from EC (Notebook 7 beta blog post #24 (comment)):
       - Today during office hours EC reviewed Notebook 7 blogpost draft (https://hackmd.io/6kExBNFcRZCIpBtQDrB0ow?both), made edits, and added comments. They liked the overall message and tone (it was pointed out that adding a bit more of "marketing" language might be beneficial though). They also asked us to go over the blogpost again cleaning the grammar, adding links, and taking comments into account. To publish the Medium from the Jupyter account instead of a personal account, we should email the request to do so to @blink1073 and @Ruv7.
       - From my point of view, blogpost should be published the same day / next day after Notebook 7 release so we should review and try to finalize it now.

- Updating the new issue templates on the repo for better a reporting experience. we stopped accepting issues through the transition. we are in a place to start accepting feedback so we should update the identity.

**Comment by jtpio** [comment]
## June 28, 2023

| name | org | username |
|------|-----|----------|
| Afshin T. Darian  | QuantStack | @afshin | 
| Andrii Ieroshenko | AWS | @andrii-i | 
| Eric Gentry | Anaconda | @ericsnekbytes  | 
| Nicolas Brichet | QuantStack | @brichet | 
|  | |  |

- Jeremy (not attending):
    - will do the 7.0 final release on July 19th
    - should we start the notebook call at 8 am Pacific on July 19th to give more time to do the release?
    - help welcome for:
        - communicating
            - Discourse thread for help with imminent 7 release [`here`](https://discourse.jupyter.org/t/new-notebook-7-release-is-coming-help-needed/20073)
            - Reddit thead [`here`](https://www.reddit.com/r/Jupyter/comments/14gaffa/new_notebook_7_release_is_coming_help_needed/)
        - monitoring issues on the repos (more people have been trying Notebook 7 recently)
        - preparing the blog post on Medium from the Jupyter account
        - 
- Nicolas:
    - nbgrader: compatibility with Notebook 7 https://github.com/jupyter/nbgrader/pull/1684

- Tony:
  - its hard to report keyboard problems with the editor but a lot of things violate expectation.
    - https://github.com/jupyter/notebook/issues/6945
    - https://github.com/jupyter/notebook/issues/6800

**Comment by jtpio** [comment]
## July 5, 2023

| name | org | username |
|------|-----|----------|
| Afshin T. Darian  | QuantStack | @afshin | 
| Eric Gentry | Anaconda | @ericsnekbytes |
| Rosio Reyes | Anaconda | @RRosio  |
| tonyfast||@tonyfast|

- Tony 
  - collect keyboard issues: https://github.com/jupyter/notebook/issues/6596
  - pre-release
- Eric working on a guide for extension capability

**Comment by jtpio** [comment]
## July 19, 2023

| name | org | username |
|------|-----|----------|
| Afshin T. Darian  | QuantStack | @afshin | 
| Eric Gentry | Anaconda | @ericsnekbytes |
| Johan Mabille | QuantStack | @JohanMabille |
| Frederic Collonval | QuantStack | @fcollonval |
| Jeremy Tuloup | QuantStack | @jtpio |

- Notebook 7 is released :tada: https://github.com/jupyter/notebook/releases/tag/v7.0.0

**Comment by jtpio** [comment]
## July 26, 2023

| name | org | username |
|------|-----|----------|
| Jeremy Tuloup | QuantStack | @jtpio |
| Afshin T. Darian | QuantStack | @afshin |
| Andrii Ieroshenko | AWS | @andrii-i |
| Eric Gentry | Anaconda | @ericsnekbytes |

- Notebook 7 blog post: https://blog.jupyter.org/announcing-jupyter-notebook-7-8d6d66126dcf

- Jeremy
    - Post 7.0 release
    - Help wanted :pray: 
        - Update `repo2docker` / Binder to JupyterLab 4 and Notebook 7: https://github.com/jupyterhub/repo2docker/pull/1302
        - Notebook 7 issues:
            - Ready for review and merge: https://github.com/jupyter/notebook/pull/6980
            - Docs: https://github.com/jupyter/notebook/issues/6976
            - MacOS only? https://github.com/jupyter/notebook/issues/6974
            - JupyterLab issue: https://github.com/jupyter/notebook/issues/6975
        - Close old issues: https://github.com/jupyter/notebook-team-compass/issues/27

- Eric G
    - JupyterCon vids status?
    - Cookiecutter --> copier?

**Comment by jtpio** [comment]
## August 09, 2023

| name | org | username |
|------|-----|----------|
| Eric Gentry | Anaconda | @ericsnekbytes |
| tonyfast| | @tonyfast |
| Andrii Ieroshenko | AWS | @andrii-i |
| Carlos Brandt | Constructor University | @chbrandt |
| Rosio Reyes | Anaconda | @RRosio |
| | | |

- Jeremy (might not make it)
    - Help troubleshoot https://github.com/jupyter/notebook/issues/6974
        - Opened https://github.com/jupyter/notebook/pull/7005 to investigate on CI
        - If someone with a Mac and using homebrew could help that would be great, thanks :pray: 
    - Reinstate the Office Hours?
        - Pinged the Notebook Council team (@jupyter/notebook-council) in https://github.com/jupyter/notebook-team-compass/issues/13#issuecomment-1665526357
        - Please leave a :-1:, :+1: or comment

- Eric G
    - Trying to merge Dual Compatibility Guide document (4 PRs)
    - Doc PR: https://github.com/jupyterlab/jupyterlab/pull/14838
        - (Example 1 PR) Toparea widget: https://github.com/jupyterlab/extension-examples/pull/240
        - (Example 2 PR) Shout button widget: https://github.com/jupyterlab/extension-examples/pull/243
        - (Example 3 PR) Third example PR coming, Clap Widget

- Andrii
    - Opinions on having a call and/or dedicated time during notebook call for triaging/closing old issues https://github.com/jupyter/notebook-team-compass/issues/27#issuecomment-1671730288

**Comment by jtpio** [comment]
## August 23, 2023

| name | org | username |
|------|-----|----------|
| Eric Gentry | Anaconda | @ericsnekbytes |
| Andrii Ieroshenko | AWS | @andrii-i |
| Jeremy Tuloup | QuantStack | @jtpio |
| | | |

We cancelled this meeting because we didn't have quorum or updates.

**Comment by jtpio** [comment]
## August 30, 2023

| name | org | username |
|------|-----|----------|
| Eric Gentry | Anaconda | @ericsnekbytes |
| Andrii Ieroshenko | AWS | @andrii-i |
| Rosio Reyes | Anaconda | @RRosio  |
| | | |
| | | |

- Eric G
    - Documentation Working Group [proposal is here](https://github.com/jupyterlab/team-compass/issues/212)!
    - We need you! :)

**Comment by jtpio** [comment]
## October 4, 2023

| name | org | username |
|------|-----|----------|
| Andrii Ieroshenko | AWS | @andrii-i |
| Eric Gentry | Anaconda | @ericsnekbytes |
| Rosio Reyes | Anaconda | @RRosio  |
| Afshin T. Darian | QuantStack | @afshin |
| Carlos Brandt | CUB | @chbrandt |
| Mike | Quansight | @krassowski |

- Darian: virtual scrollbar PR https://github.com/jupyterlab/jupyterlab/pull/15109

**Comment by jtpio** [comment]
## October 11, 2023

| name | org | username |
|------|-----|----------|
| Eric Gentry | Anaconda | @ericsnekbytes |
| Rosio Reyes | Anaconda | @RRosio  |
| Carlos Brandt | CUB | @chbrandt |
| Mike | Quansight | @krassowski |
| Andrii Ieroshenko | AWS | @andrii-i |
| Jeremy Tuloup | QuantStack | @jtpio |

**Comment by jtpio** [comment]
## October 18, 2023

| name | org | username |
|------|-----|----------|
| Eric Gentry | Anaconda | @ericsnekbytes |
| Jeremy Tuloup | QuantStack | @jtpio |

Docs discussion: https://github.com/jupyter/docs-team-compass/issues/14

**Comment by jtpio** [comment]
## October 25, 2023

| name | org | username |
|------|-----|----------|
| Eric Gentry | Anaconda | @ericsnekbytes |
| Jeremy Tuloup | QuantStack | @jtpio |
| Andrii Ieroshenko | AWS | @andrii-i |

### Agenda

Various discussions about how extensions can know more about the app they are being run in.

**Comment by jtpio** [comment]
## November 1, 2023

| name | org | username |
|------|-----|----------|
| Eric Gentry | Anaconda | @ericsnekbytes |
| Andrii Ieroshenko | AWS | @andrii-i |
| Mike | Quansight | @krassowski |
| | | |
| | | |
| | | |
| | | |

### Agenda

- JupyterLite
    - Use lite to test deep stacks of Jupyter dependencies (lite, notebook, lab, lumino)
    - Need to be able to drop ipython in and get something useful
    - Whats possible for us to build with web platform? Docker not viable given lack of resources
    - Change something in lumino, see what it does in ipywidgets

**Comment by jtpio** [comment]
## November 8, 2023

| name | org | username |
|------|-----|----------|
| Eric Gentry | Anaconda | @ericsnekbytes |

- No agenda items

**Comment by jtpio** [comment]
## November 15, 2023

| name | org | username |
|------|-----|----------|
| Eric Gentry | Anaconda | @ericsnekbytes |
| Andrii Ieroshenko | AWS | @andrii-i |
| Carlos Brandt | CUB | @chbrandt |
| Rosio Reyes | Anaconda | @RRosio  |

- Andrii: discussion on best practices of passing error from jupyter server backend to frontend
- Tony: running a community call over notebook and lab meetings on dec 13? 
  - Notebook team agreed

**Comment by jtpio** [comment]
## November 22, 2023

No meeting

**Comment by jtpio** [comment]
## November 29, 2023

| name | org | username |
|------|-----|----------|
|Frederic Collonval | QuantStack | @fcollonval |
| Andrii Ieroshenko | AWS | @andrii-i |
| Jeremy Tuloup | QuantStack | @jtpio |
|Carlos Brandt | CUB | @chbrandt |
| Rosio Reyes | Anaconda | @RRosio  |

### Agenda

- Eric G
    - Future direction of Notebook 7  
      Some discussion issues about it  
        https://github.com/jupyterlab/team-compass/issues/80  
        https://github.com/jupyter/notebook/issues/7134
      [Nick] standards compliance  
        Get off of specific techno like webpack to use more standards  
        Have more things backed by data models

- Frederic
    - Should we merge this meeting with the JupyterLab one as now the
      work on Notebook 6 is done and Notebook 7 shares code with JupyterLab 4?

- Rosio
    - Two questions from the triage meeting
        - Customized magics? https://github.com/jupyter/notebook/issues/7158
        - https://github.com/jupyter/notebook/issues/7163
            - CentOS file names with underscores or dashes/hyphens?

**Comment by jtpio** [comment]
## December 6, 2023

| name | org | username |
|------|-----|----------|
| Jeremy Tuloup | QuantStack | @jtpio |
| Eric Gentry | Anaconda | @ericsnekbytes |
| Carlos Brandt | CUB | @chbrandt |
| Rosio Reyes | Anaconda | @RRosio  |
| Andrii Ieroshenko | AWS | @andrii-i |

- Jeremy:
    - New Notebook 7.1 pre-release: https://github.com/jupyter/notebook/releases/tag/v7.1.0a1
    - Include the latest additions from JupyterLab 4.1 such as the inline code completion API: https://github.com/jupyter/notebook/pull/7161#issuecomment-1833885558

- Rosio
    - Issue running notebook: ["Notebook doesn't open when jupyter server runs on pypy3"](https://github.com/jupyter/notebook/issues/7171)

**Comment by jtpio** [comment]
Closing as the proposal for merging the JupyterLab and Notebook subprojects has been accepted: https://github.com/jupyter/governance/pull/200
